packages/strapi-admin/admin/src/plugins.js

```diff
@@ -27,14 +27,11 @@ module.exports = {
     .default,
   'strapi-plugin-content-manager': require('../../../strapi-plugin-content-manager/admin/src')
     .default,
-  'strapi-plugin-content-type-builder': require('../../../strapi-plugin-content-type-builder/admin/src')
-    .default,
-  'strapi-plugin-documentation': require('../../../strapi-plugin-documentation/admin/src')
-    .default,
-  'strapi-plugin-email': require('../../../strapi-plugin-email/admin/src')
-    .default,
-  'strapi-plugin-upload': require('../../../strapi-plugin-upload/admin/src')
-    .default,
-  'strapi-plugin-graphql': require('../../../strapi-plugin-graphql/admin/src')
-    .default,
+  // 'strapi-plugin-content-type-builder': require('../../../strapi-plugin-content-type-builder/admin/src')
+  //   .default,
+  // 'strapi-plugin-documentation': require('../../../strapi-plugin-documentation/admin/src')
+  //   .default,
```

Comment:
Well in development I tend to remove of the plugins to reduce the reload time I 'll put them back but it does not have an impact on the generated app since this file is automatically created.